jira/client.py

```diff
@@ -2949,7 +2949,7 @@ def create_project(self, key, name=None, assignee=None, type="Software", templat
         return False
 
     def add_user(self, username, email, directoryId=1, password=None,
-                 fullname=None, notify=False, active=True, ignore_existing=False):
+                 fullname=None, notify=False, active=True, ignore_existing=False, applicationKeys=None):
```

Comment:
I'd argue that applicationKeys should use snake case and be `application_keys`.

Reply:
Good call. I've updated the argument.  I had it consistent with the api call, but it makes more sense to be consistent with the rest of the package.